src/pgduckdb_options.cpp

```diff
@@ -296,6 +297,60 @@ DECLARE_PG_FUNCTION(pgduckdb_raw_query) {
 	PG_RETURN_BOOL(true);
 }
 
+DECLARE_PG_FUNCTION(pgduckdb_is_motherduck_enabled) {
+	PG_RETURN_BOOL(pgduckdb::IsMotherDuckEnabled());
+}
+
+DECLARE_PG_FUNCTION(pgduckdb_enable_motherduck) {
+	if (pgduckdb::IsMotherDuckEnabled()) {
+		elog(NOTICE, "MotherDuck is already enabled");
+		PG_RETURN_BOOL(false);
+	}
+
+	auto token = pgduckdb::pg::GetArgString(fcinfo, 0);
+	auto default_database = pgduckdb::pg::GetArgString(fcinfo, 1);
+
+	// If no token provided, check that token exists in the environment
+	if (token == "::FROM_ENV::" && getenv("MOTHERDUCK_TOKEN") == nullptr && getenv("motherduck_token") == nullptr) {
+		elog(ERROR, "No token was provided and `motherduck_token` environment variable was not set");
+	}
+
+	SPI_connect();
+
+	if (pgduckdb::GetMotherduckForeignServerOid() == InvalidOid) {
+		std::string query = "CREATE SERVER md_server TYPE 'motherduck' FOREIGN DATA WRAPPER pg_duckdb";
```

Comment:
I think it's nicer to just call this `motherduck`.
```suggestion
		std::string query = "CREATE SERVER motherduck TYPE 'motherduck' FOREIGN DATA WRAPPER pg_duckdb";
```

Reply:
IIRC we can't name it `motherduck` because it is already a known symbol, let me try again.

Reply:
Well I take that back, it seems to work. Not sure what I did before...

Reply:
maybe you used single quotes around it ;)